app/components/chart/trends.js

```diff
@@ -35,6 +36,7 @@ const TrendsContainer = viz.containers.TrendsContainer;
 const reshapeBgClassesToBgBounds = viz.utils.reshapeBgClassesToBgBounds;
 const Loader = viz.components.Loader;
 
+@translate()
```

Comment:
Since we currently don't have any decorator patterns in the codebase, I'd be hesitant to add it right now. I'm not opposed to the idea, but perhaps I'd like @cbwebdevelopment's opinion as well.

Reply:
I agree.  I actually do like decorator patterns (I've used them on past projects), but for consistency with the rest of the codebase at this point, I think we should hold off on introducing them, and use the HOC approach used everywhere else.

Reply:
I've updated the PR by removing the decorator pattern. I also marked the rest of blip for translation (all tests passing) in the  `language` branch of my fork, ready to make another PR.